plugins/outputs/splunk/splunk.go

```diff
@@ -0,0 +1,298 @@
+package splunk
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+type Splunk struct {
+	Prefix string
+	Source string
+
+	SplunkUrl  string
+	AuthString string
+	Timeout    internal.Duration
+	client     *http.Client
+
+	SimpleFields    bool
+	MetricSeparator string
+	ConvertPaths    bool
+	ConvertBool     bool
+	ReplaceSpecials bool
+	UseRegex        bool
+	StringToNumber  map[string][]map[string]float64
```

Comment:
The response to the 'lack of documentation' comment on the previous review said documentation had been added on this parameter. But I'm still not seeing it. Maybe it was accidentally left out of the update?

Reply:
The documentation was added to the readme and the notes in the config.   I've add comments to the code as well.